app/ui-message/client/messageBox/userActionIndicator.html

```diff
@@ -0,0 +1,7 @@
+<template name="userActionIndicator" args="rid, tmid">
+	<div class="rc-message-box__activity-indicator">
+		{{> notifyUserActivity data=typingUsersData}}
```

Comment:
If you accept my suggestion to simplify you can eliminate this file since the render will be dynamic in a look on the other template